### PR TITLE
Add survey-vers.json

### PR DESCRIPTION
**Pull request by jayoung-lee**
This is a json file for future quarterly surveys.

### CODE REVIEW

File: src/_data/survey-vers.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "uniqueId": "2020Q1",
+  "startMsEpoch": "2020/03/03 9:00",
```
Comment (devoncarew) [inline src/_data/survey-vers.json]: `startDate`?

And, I'd make sure that the format here aligns with something in https://api.dart.dev/stable/2.7.1/dart-core/DateTime/parse.html.

Perhaps just ensure that the value always looks like `"2012-02-27 13:27:00"`?

File: src/_data/survey-vers.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "endMsEpoch": "2020/03/10 9:00",
```
Comment (devoncarew) [inline src/_data/survey-vers.json]: `endDate`?

File: src/_data/survey-vers.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "title": "Help improve Flutter! Take our Q1 survey.",
```
Comment (devoncarew) [inline src/_data/survey-vers.json]: I'd add comments to the file for this field, to indicate that the title has to be super brief; we're limited in IDEs as to how much we can display. Really, the current string is the max length.

File: src/_data/survey-vers.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  "urlPrefix": "https://google.qualtrics.com/jfe/form/SV_6YBXYxIR69pgpr7"
```
Comment (devoncarew) [inline src/_data/survey-vers.json]: `url`? IDEs will know to add any additional url parameters

**Comment by devoncarew** [comment]
Additionally, perhaps rename from `src/_data/survey-vers.json` to something like `src/_data/survey-metadata.json`? `src/_data/flutter-survey.json`?

**Comment by devoncarew** [comment]
@sfshaza2, @legalcodes - how frequently are changes pushed to the website? Or, what would the typical latency be between changes landing in the file on github and the changes being live on flutter.dev?

**Comment by legalcodes** [comment]
@devoncarew The typical latency between changes landing on GitHub and live on flutter.dev is ~10minutes.

**Comment by jayoung-lee** [comment]
FYI, this PR is reverted. The new PR that makes this file accessible externally is https://github.com/flutter/website/pull/3672

I think @devoncarew 's comments are all valid. @pq and @DanTup, let me know which format you'd like.

(For your context @DanTup, we are trying to host a json file online so that you don't have to push the change every quarter. @pq will land his PR to IntelliJ soon and cc you.)

**Comment by devoncarew** [comment]
Great! Sounds good. If it was on the order of a few weeks we would have needed to plan for that wrt changes to survey dates and such.

**Comment by DanTup** [comment]
Sgtm! I added a comment on #3672 along the same lines as Devon about date formats (specifically timezones and a standard format so it's easier to parse in different languages).

**Pull request merged**